inc/mlx90632.h

```diff
@@ -239,6 +239,31 @@ double mlx90632_calc_temp_object(int32_t object, int32_t ambient,
                                  int32_t Ea, int32_t Eb, int32_t Ga, int32_t Fa, int32_t Fb,
                                  int16_t Ha, int16_t Hb);
 
+/** Calculation of object temperature when the environment temperature differs from the sensor temperature
+ *
+ * DSPv5 implementation of object temperature calculation with customer
+ * calibration data
+ *
+ * @param[in] object object temperature from @link mlx90632_preprocess_temp_object @endlink
+ * @param[in] ambient sensor ambient temperature from @link mlx90632_preprocess_temp_ambient @endlink
+ * @param[in] reflected reflected (environment) temperature from a sensor different than the MLX90632 or acquired by other means
```

Comment:
```suggestion
 * @param[in] reflected Environment temperature from a sensor different than the MLX90632 or acquired by other means
```

Reply:
I think reflected is the right term. And this could also be acquired using our sensor.